tox.ini

```diff
@@ -8,7 +8,7 @@ envlist =
 skip_missing_interpreters = true
 
 [pytest]
-addopts = --strict-markers
+addopts = --strict-markers --verbose --disable-warnings --cache-clear --maxfail=3 -p no:doctest
```

Comment:
Curious why tests should stop after 3 failures? In a bunch of tests failed in CI, we'd only get to see the first three fail, right?

http://doc.pytest.org/en/latest/usage.html

Reply:
Also - is there any specific reason for disabling warnings?
[Docs](https://docs.pytest.org/en/latest/warnings.html?highlight=disable-warning#disabling-warnings-summary) doesn't recommend disabling it (I guess, because this is a global option).

Reply:
👍 Often times the warnings (especially `DeprecationWarning`s) are useful.

Reply:
I made bad experiences with Tensorflow and warnings ... so I just copied the command. So I will change it.